srcpkgs/kore/patches/python-3.11.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/src/python.c b/src/python.c
+index ab0f3c6..526d443 100644
+--- a/src/python.c
++++ b/src/python.c
+@@ -1192,8 +1192,14 @@ python_coro_trace(const char *label, struct python_coro *coro)
+ 
+ 	gen = (PyGenObject *)coro->obj;
+ 
+-	if (gen->gi_frame != NULL && gen->gi_frame->f_code != NULL) {
+-		code = gen->gi_frame->f_code;
++#if PY_VERSION_HEX < 0x030B0000
++	code = NULL;
++	if (gen->gi_frame != NULL)
++		code = gen->gi_frame->f_code;
++#else
++	code = gen->gi_code;
++#endif
++	if (code != NULL) {
+ 		func = PyUnicode_AsUTF8AndSize(code->co_name, NULL);
+ 		file = PyUnicode_AsUTF8AndSize(code->co_filename, NULL);
+ 
+@@ -1206,8 +1206,13 @@ python_coro_trace(const char *label, struct python_coro *coro)
+ 		fname = "unknown";
+ 	}
+ 
++#if PY_VERSION_HEX <= 0x030B0000
```

Comment:
It turns out that my previous recommendation for strictly less than is irrelevant because the release version will be `0x030B0F00` and the current RC should be `0x030B0C02`, so both are greater than the cutoff anyway. I misread the explanation of the hex versioning scheme and expected the release to be 0x30B00000`.

I'd say make this strictly less than for consistency and mathematical correctness.